### PR TITLE
Adds new magazine/drum sprites + code fixes / additions

### PR DESCRIPTION
**Pull request by AMcilraith**
## About The Pull Request
<details>
<summary>
Adds new sprites for ammunition magazines and containers.
</summary>
<hr><br>
This PR is my unabashed attempt at spriting in the following variants for magazines of varying kinds, along with adding the code to spawn/use if it's ever decided these are wanted later. Included are: <br>
- SI LASER magazines, for each provided caliber. Added to code in the event future use is needed.<br>
- Pepperball magazines, now distinctly different from hollow point mags. Added new iconstates to code.<br>
- Fixes to some of the High-Velocity mags that were not previously green. <br>
- Changing incendiary magazines/drums to white.<br>
- CBO sleeper rounds drum is now legally distinct from rubbers. Added new iconstates to code.<br>
<hr>
</details>

## Changelog
:cl:
add: SI LASER ammo magazines (code and sprites)
add: new pepperball ammo magazines (sprites)
add: new CBO ammo drum (sprites)
fix: fixed inconsistent green-ness on hv mag sprites (sprites)
fix: changed incendiary to be white lines (sprites)
Trilby Edits:
Fixes inconstant naming in bullet fab
/:cl:

## Screenshots
**SI LASER Magazines**
image
**New Pepperball Magazines**
image
**Consistent High Velocity Magazines**
image
**Consistent Incendiary Magazines/Drums**
image
**New CBO Sleeper Drum**
image

### CODE REVIEW

File: code/modules/projectiles/ammunition/magazines.dm [hunk 1]
```diff
@@ -1195,13 +1195,83 @@
...
+
+/obj/item/ammo_magazine/magnum_40/laser/empty
+	desc = "A 10 round magazine marked for 10mm Magnum. This one is not pre-filled, and is intended for SI LASER ammunition."
+	initial_ammo = 0
+
+/obj/item/ammo_magazine/kurtz_50/laser
+	name = "heavy pistol magazine (12mm LASER)"
+	desc = "A 7 round magazine marked for 12mm. This one is intended for SI LASER ammunition."
+	icon_state = "kurtz_lsr"
+	ammo_type =
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/ammunition/magazines.dm:1237]: this is missing ammo type

Comment (AMcilraith) [reply code/modules/projectiles/ammunition/magazines.dm]: yeah, i was fixing it as you wrote it. it SHOULD be fixed now.

Comment (AMcilraith) [reply code/modules/projectiles/ammunition/magazines.dm]: _/obj/item/ammo_magazine/ammobox/kurtz_50/laser_
is the correct path

Comment (AMcilraith) [reply code/modules/projectiles/ammunition/magazines.dm]: actually no
it's /obj/item/ammo_casing/kurtz_50/laser and i was totally wrong
fixing now

**Comment by AMcilraith** [comment]
@Trilbyspaceclone tests seem to run fine on my end, getitng no syntax errors.

when the checks (probably, hopefully) pass, can we at least remove the "GAME BREAKING" tag?

**Comment by AMcilraith** [comment]
A moment, got a request on the discord to fix an error that's already present and not fixed by this PR. I'll do my best to work on it to see what causes it first, but in the meantime, hold off on merging.

**Pull request closed without merging**